src/pyaro_readers/eeareader/EEATimeseriesReader.py

```diff
@@ -73,6 +78,7 @@ def __init__(
         self._set_filters(filters)
 
         self.metadata = self._read_metadata(filename)
+        self.datafile = self._read_data()
```

Comment:
both `datafile` and `_read_data` and `data.toml` are very confusing here. It is everything but `data`=measurement of e.g. 17.35µg/m³, please find a better name. `cfg` or `setup`? And definitely don't call something for `file` which is neither a filename or filehandle.